packages/gatsby/src/utils/cache-lmdb.ts

```diff
@@ -16,12 +16,20 @@ const cacheDbFile =
 export default class GatsbyCacheLmdb {
   private static store
   private db: Database | undefined
+  private readonly encoding: DatabaseOptions["encoding"]
   public readonly name: string
   // Needed for plugins that want to write data to the cache directory
   public readonly directory: string
 
-  constructor({ name = `db` }: { name: string }) {
+  constructor({
+    name = `db`,
+    encoding = `msgpack`,
```

Comment:
From https://github.com/gatsbyjs/gatsby/pull/32466

Should this be `json` instead?

```suggestion
    encoding = `json`,
```